cmd/ejson2env/secrets.go

```diff
@@ -48,6 +48,10 @@ func exportSecrets(filename, keyDir, privateKey string) error {
 		return fmt.Errorf("could not load environment from file: %s", err)
 	}
 
-	ExportEnv(os.Stdout, envValues)
+	if quiet {
```

Comment:
If we were using the final argument as the method, rather than a boolean flag, this would then become:

```golang
exportFunc(os.Stdout, envValues)
return nil
```

The big advantage of this is that we will be able to continue to add new Export functions without changing this method.